src/js/EditsContainer.jsx

```diff
@@ -23,7 +23,7 @@ var EditsContainer = React.createClass({
   },
 
   propTypes: function(){
-    setAppStatus: React.PropTypes.func
+    appStatus: React.PropTypes.object
```

Comment:
Could we also define that the object should contain functions?
https://facebook.github.io/react/docs/reusable-components.html

```javascript
appStatus: React.PropTypes.objectOf(React.PropTypes.func)
```